similar/editor/med.cpp

```diff
@@ -856,7 +880,8 @@ static void close_editor()
 
 	ui_close();
 
-	editor_font.reset();
+	if (editor_font)
```

Comment:
This test is unnecessary.  `std::unique_ptr` (which is used for the implementation of `grs_font_ptr`) already has a test internally to detect `nullptr` and skip freeing it.